### PR TITLE
RK-14604 - Add JBoss and Wildfly to whitelist required list

### PR DESCRIPTION
**Pull request by naomirookout**

### CODE REVIEW

File: docs/jvm-setup.mdx [hunk 1]
```diff
@@ -66,11 +66,11 @@ import TabItem from '@theme/TabItem';
 
 <div className="rookout-org-info" />
 
-#### Debugging OSGi apps
+#### Whitelisting Rookout
```
Comment (LiranLast) [inline docs/jvm-setup.mdx:69]: "Whitelisting" is about the how, not what and why :)

File: docs/jvm-setup.mdx [hunk 2]
```diff
@@ -66,11 +66,11 @@ import TabItem from '@theme/TabItem';
...
 
-To use Rookout in an OSGi application, the Rookout package must be whitelisted (delegated) to the OSGi container.
+In OSGi, JBoss and Wildfly apps, Rookout must be whitelisted in order to run.
```
Comment (LiranLast) [inline docs/jvm-setup.mdx:71]: JBoss and Wildfly are examples of OSGi

File: docs/jvm-setup.mdx [hunk 3]
```diff
@@ -80,6 +80,8 @@ org.osgi.framework.bootdelegation=com.rookout.*
 
 This is done where all other JVM options are set (via -D flags, JVM_OPTS, etc.).
 
+You can see our [JBoss](https://github.com/Rookout/deployment-examples/tree/master/java-jboss) and [Wildfly](https://github.com/Rookout/deployment-examples/tree/master/java-wildfly-docker) deployment examples to learn how to whitelist Rookout in those applications.
```
Comment (LiranLast) [inline docs/jvm-setup.mdx:83]: We don't link examples in the core docs, they are not usually well enough maintained

**Comment by naomirookout** [comment]
@LiranLast I'm closing this PR, it looks like none of these changes are needed.

**Comment by LiranLast** [comment]
> @LiranLast I'm closing this PR, it looks like none of these changes are needed.

I think we can definitely improve, let's just think about it together :)

**Pull request closed without merging**